NotesApi/Startup.cs

```diff
@@ -115,12 +122,33 @@ private static void ConfigureDbContext(IServiceCollection services)
             var connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING");
 
             services.AddDbContext<DatabaseContext>(
-                opt => opt.UseNpgsql(connectionString));
+                opt => opt.UseNpgsql(connectionString).AddXRayInterceptor(true));
+        }
+
+        private static void ConfigureLogging(IServiceCollection services, IConfiguration configuration)
+        {
+            // We rebuild the logging stack so as to ensure the console logger is not used in production.
+            // See here: https://weblog.west-wind.com/posts/2018/Dec/31/Dont-let-ASPNET-Core-Default-Console-Logging-Slow-your-App-down
+            services.AddLogging(config =>
+            {
+                // clear out default configuration
+                config.ClearProviders();
+
+                config.AddConfiguration(configuration.GetSection("Logging"));
+                config.AddDebug();
+                config.AddEventSourceLogger();
+
+                if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == Environments.Development)
+                {
+                    config.AddConsole();
+                }
+            });
         }
 
         private static void RegisterGateways(IServiceCollection services)
         {
             services.AddScoped<INotesApiGateway, NotesApiGateway>();
```

Comment:
This line should be deleted.
And if it still exists, the `NotesApiGateway` class should also be deleted, as it is Postgres-specific.